cms/sass/pages/_homepage.scss

```diff
@@ -13,8 +13,8 @@
 
   .primary-nav--scrolled {
     background-color: rgba($light-grey, 0.5);
-    -webkit-backdrop-filter: blur(8px);
-    backdrop-filter: blur(8px);
+    -webkit-backdrop-filter: blur(0.5rem);
+    backdrop-filter: blur(0.5rem);
```

Comment:
Is it correct to apply the `rem` approach to visual elements like blur, which don't have a textual component?